project/sites/victimspaymentsboard/themes/victimspaymentsboard_theme/src/scss/components.scss

```diff
@@ -98,9 +98,12 @@ h1 {
     border-bottom: none;
     color: #0e6bb3;
 
-      &:hover {
-        text-decoration: underline;
+    span {
+      border-bottom: 1px solid #0e6bb3;
 
+      &:hover {
+        border-bottom: 2px solid #000;
+      }
```

Comment:
Should really use variables in this file as they are already declared.
`border-bottom: 1px solid $link` and `border-bottom: 2px solid $black`

Reply:
Changes made. Thanks Adam